### ISSUE TITLE
FEBio 3 does not recognize FEBio input spec 2.5 node-varying boundary conditions

### ISSUE THREAD

**Issue by jpeloquin** [febiosoftware/FEBio#25]
FEBio 3.1.0 (the Nov 23 release from https://febio.org/downloads/) does not recognize prescribed node-varying boundary conditions in an XML file that conforms to FEBio input specification 2.5.  According to the FEBio 3.0 manual, FEBio 3.0 is still supposed to support FEBio input specification 2.5 ("This format is still supported but considered obsolete.").

For example, a file with an `<febio_spec version="2.5">` declaration that contains

```xml
<Boundary>
      <prescribe bc="z" node_set="node_set_bc_x3">
        <scale lc="2">1.0</scale>
        <value node_data="node_data_bc_x3"/>  <!-- this is the line 703 referred to in the subsequent error -->
        <relative>0</relative>
      </prescribe>
    </Boundary>
```

when run in FEBio 3.1.0 produces the error `tag "value" (line 703) : unrecognized tag`.

Based on Issue #18, I gather that lack of support for input specification 2.5 might be considered normal for FEBio 3.  If so, please consider this issue as a request to revise the manual to reflect that FEBio 3 only runs input specification 3.0, or otherwise clarify what the limits of backwards compatibility are for FEBio 3.

I have no objection to a clean break in compatibility if that makes development easier.  Mostly I just need to know whether I can drop support for input specification 2.5 in my own work.  If FEBio 3 can't read it, it's functionally dead, as FEBio 3 has critical bug fixes vs. FEBio 2.

**Comment by SteveMaas1978** [comment]
Hi @jpeloquin,

In the latest release (FEBio 3.2, released last Friday, Feb 5), some backward compatibility issues with the 2.5 format were addressed, including this one. Please try this newer version and let us know if your file now runs. 

Although we recommend that new models use the 3.0 format, we definitely try to continue supporting the 2.5 format. I hope that we have addressed any backward compatibility issues by now, but let us know if you find any other.

**Comment by jpeloquin** [comment]
Thanks Steve.  That's a spot of good news.

Issue confirmed fixed with FEBio 3.2.
